doc/vim-lsp.txt

```diff
@@ -625,9 +625,18 @@ The vim |dict| containing information about the server.
 	'whitelist': ['*']
 	'blacklist': ['javascript', 'typescript']
 
+  * workspace_config:
+    optional vim |dict|
+    Used to pass workspace configuration to the server after
+    initialization. Configuration settings are language-server specific.
+
+    Example:
+	'workspace_config': {'pyls': {'plugins': \
+		{'pydocstyle': {'enabled': v:true}}}}
+
   * config:
     optional vim |dict|
-    Used to pass additional custom config.
+    Used to pass additional custom config. See |vim-lsp-config-server|.
```

Comment:
Can you just add the `vim-lsp-config-server` section (or remove the reference)?

Reply:
Oops, forgot to remove that. (But I can also add the section -- although it's very close to that location. What would you prefer?)

Reply:
Let's just remove it, otherwise we also have to add an entry to the contents.